extensions/bitwarden/src/search.tsx

```diff
@@ -225,3 +226,17 @@ function PasswordActions(props: { password: string }) {
     <React.Fragment>{primaryAction == "copy" ? [copyAction, pasteAction] : [pasteAction, copyAction]}</React.Fragment>
   );
 }
+
+function UsernameActions(props: { username: string }) {
+  return (
+    <ActionPanel.Item
+      shortcut={{ modifiers: ["cmd"], key: "u" }}
+      title="Copy Username"
+      icon={Icon.Person}
+      onAction={() => {
+        Clipboard.copy(props.username);
+        closeMainWindow({ clearRootSearch: true });
+      }}
+    />
+  );
+}
```

Comment:
Please use the `Action.CopyToClipboard`. You can then use the `onCopy` prop to close the main window.

Reply:
Thanks for the feedback @pomdtr, this has been actioned.